package.json

```diff
@@ -1,15 +1,24 @@
 {
   "name": "reactor-iota",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "An EVRYTHNG Blockchain Integration Hub reactor script for IOTA",
   "main": "main.js",
-  "author": "EVRYTHNG",
+  "authors": [
+    "EVRYTHNG",
+    "IOTA Foundation"
+  ],
   "license": "Apache-2.0",
+  "scripts": {
+    "lint": "semistandard *.js"
```

Comment:
We use `eslint-config-airbnb-base` for our projects, let's use it here too.

https://www.npmjs.com/package/eslint-config-airbnb-base

Reply:
fixed!